src/games/supported/BreakoutSuper.cpp

```diff
@@ -0,0 +1,177 @@
+/* *****************************************************************************
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License version 2
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ * *****************************************************************************
+ * A.L.E (Arcade Learning Environment)
+ * Copyright (c) 2009-2013 by Yavar Naddaf, Joel Veness, Marc G. Bellemare and
+ *   the Reinforcement Learning and Artificial Intelligence Laboratory
+ * Released under the GNU General Public License; see License.txt for details.
+ *
+ * Based on: Stella  --  "An Atari 2600 VCS Emulator"
+ * Copyright (c) 1995-2007 by Bradford W. Mott and the Stella team
+ *
+ * *****************************************************************************
+ */
+#include "BreakoutSuper.hpp"
+
+#include "../RomUtils.hpp"
+
+
+BreakoutSuperSettings::BreakoutSuperSettings() {
+
+    m_reward   = 0;
+    m_score    = 0;
+    m_lives    = 0;
```

Comment:
@pvchaumier , why did you define m_lives as the number of lives you lost instead of the number of lives you currently have? This seems counter-intuitive that you have to return 6 - m_lives and that here, and in the reset function you set m_lives to be zero, but them you update it at every step... Why don't you just set it to 6 here and in reset?